.github/workflows/stacks-node-image.yml

```diff
@@ -0,0 +1,81 @@
+name: Publish stacks-node testing image
+
+on:
+  workflow_dispatch:
+    inputs:
+      stacks_blockchain_branch:
+        description: 'stacks-blockchain git branch or tag'
+        required: true
+        type: string
+
+env:
+  STACKS_BLOCKCHAIN_BRANCH: ${{ inputs.stacks_blockchain_branch || 'next' }}
+
+jobs:
+  build-stacks-node:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
Seems like this step can be used to checkout the stacks-blockchain repo instead of the api repo

Reply:
It checks out the API repo to grab the dockerfile (in this repo) used in a job later down. That _could_ be changed to pull the dockerfile via a git uri but the syntax isn't very nice.